### ISSUE TITLE
Could not unlock sender account - error

### ISSUE THREAD

**Issue by bamos01** [ethereum/mist#187]
Hi- using Mist v 0.4.0 on Mac OSx

I originally installed geth and imported a wallet from private key using command line.  Everything worked as expected and balances showing up properly in Mist.  However, if I try to send a transaction using Mist i get the error "Cound not unlock sender account" - this problem can be fixed by first running geth from the command line and then opening Mist.  This is confusing because I thought I could just open and run Mist directly rather than having to first start a geth instance from the command line.  Is this the way it is supposed to work?

**Comment by frozeman** [comment]
No it should actually work straight away..
Did you pre-unlock the account the in geth command line?

**Comment by bamos01** [comment]
No- i didn't see any instructions that I should do that- is that a necessary step when importing a wallet using a private key?  Do i need to do that only one time?

**Comment by frozeman** [comment]
No it should just work.

Which version you have started from the command line? the version bundled in geth is 1.3.3

**Comment by bamos01** [comment]
"Starting Geth/v1.3.3/darwin/go1.5.3"

**Comment by frozeman** [comment]
Thanks there shouldn't be any difference in wether mist starts it or yourself.
Can you restart and try again?

**Comment by bamos01** [comment]
Restarted computer- confirming that I am replicating following behavior:

Unless I first start geth and initiate node from command line, I receive following error when sending a transaction with mist "Cannot unlock signing account"

**Comment by frozeman** [comment]
Thats very weird. could you move the funds out of this account and send it to me including password?
fabian at ethdev.com

This way i can try out myself and see whats the issue

**Comment by bamos01** [comment]
Hi - thanks for your response. I played around with it a bit more and
realized that I was getting this error somewhat randomly and it was not
always related to whether I started geth first or not. I don't know what
was causing the problem but there was definitely an unexpected error that
was occurring intermittently about not being able to unlock sender account.
All accounts were created w geth rather than through mist so I don't know
if this was an issue. Restarting either Mist or geth seemed to temporarily
resolve the issue.
